README.md

```diff
@@ -1,36 +1,59 @@
 # Utils
 
-[![Build Status](https://travis-ci.org/kubernetes/utils.svg?branch=master)](https://travis-ci.org/kubernetes/utils)
+[![Build Status]](https://travis-ci.org/kubernetes/utils)
 
-Utils is a set of golang libraries that are not specific to
-Kubernetes. They should be available and useful to any other Go project
-out there.
+A set of Go libraries that provide low-level,
+kubernetes-independent packages supplementing the [Go
+standard libs].
 
 ## Purpose
 
-The Kubernetes project uses a lot of Golang patterns that are re-used in many
-difference places, and as the project is further split into different
-repositories, this repository is the perfect place to hold this common code.
+As Kubernetes grows and spins functionality out of its
+[core] and into cooperating repositories
+like [apiserver], [kubectl], [kubeadm], etc.,
+the need arises for leaf repos to house shared code
```

Comment:
I'm not a super fan of using "repo(s)" rather than "repository(ies)". Also, while saying that, I notice that there are ... 8 occurrences of "repo" in this section :-)